contracts/Recruitment.sol

```diff
@@ -452,6 +464,63 @@ contract Recruitment is Ownable, ReentrancyGuard {
         frontDoorToken.transfer(msg.sender, bounty);
     }
 
+    function setCanidateScoreFromCompany(address candidateAddress, uint256 score) public {
+        require(isCompany[msg.sender] == true , "You can't give score to candidate");
+        FrontDoorStructs.UserScore memory newScore = FrontDoorStructs.UserScore (
+          score,
+          msg.sender
+        );
+        candidateScores[candidateAddress].push(newScore);
+        referralScores[candidateList[candidateAddress].referrer].push(newScore);
+        updateCandidateScore(candidateAddress);
+        updateReferrerScore(candidateList[candidateAddress].referrer);
+    }
+    function setCompanyScoreFromCandidate(address companyAddress, uint256 score) public checkIfCandidateHiredByCompany(msg.sender , companyAddress) {
```

Comment:
check formatting - each function should be separated by a new line.

Reply:
Yeah, I missed that. thx.